### PR TITLE
Timestamp and BlockNumber fields are added to transactions.model.js

### PR DESCRIPTION
**Pull request by aminlatifi**
* Old transactions records will be removed by migration script
* conversionRates supports rate by transaction hash

### CODE REVIEW

**Comment by aminlatifi** [comment]
@mohammadranjbarz Unit tests are failed since transaction model is changed and test seed data is not compatible with new model. Can you fix it for this case? we can discuss later about how to save db data for test after.

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz Unit tests are failed since transaction model is changed and test seed data is not compatible with new model. Can you fix it for this case? we can discuss later about how to save db data for test after.

sure I'll do it

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz Unit tests are failed since transaction model is changed and test seed data is not compatible with new model. Can you fix it for this case? we can discuss later about how to save db data for test after.

@aminlatifi 
I had bad strategy for restoring dump for tests, it changed now, you just put a dump from `giveth-test` dbName in `test/db_seed_data `  folder, and everything will be handle ( so if you want to change something you can restore that dump change it and after that get a new dump and replace with old one)


but now tests still will fail because the transcactionModel doesn't have default value for  `isHome` , if you want to test pass you can set default value `false` for that field then tests will pass, I testedand it was ok, but I didn't change that because I think maybe logically it's false to set isHome false ( I have no idea)

File: src/blockchain/lib/web3Helpers.js [hunk 1]
```diff
@@ -160,10 +161,8 @@ const txListeners = {};
  * @param {boolean} isHome get transaction of home network
  */
 const getTransaction = async (app, hash, isHome = false) => {
-  const web3 = isHome ? app.getHomeWeb3() : app.getWeb3();
   const Transaction = app.get('transactionsModel');
-  const query = { hash };
-  if (isHome) query.isHome = true;
+  const query = { hash, isHome: !!isHome };
```
Comment (mohammadranjbarz) [inline src/blockchain/lib/web3Helpers.js:165]: Why you just didn't use `isHome` instead of `!!isHome` ?

**Pull request merged**